src/oc_erchef/apps/oc_chef_wm/src/oc_chef_associations.erl

```diff
@@ -147,33 +147,33 @@ deprovision_delete_usag(ok, #context{usag = USAG,
                                      requestor_authz_id = RequestorID } = Context) ->
     % This will delete both the USAG and its authz entry.
     Result = oc_chef_object_db:safe_delete(DbContext, USAG, RequestorID),
-    deprovision_fetch_org_global_admins(Result, Context);
+    deprovision_fetch_org_read_access_group(Result, Context);
 deprovision_delete_usag(Error, _Context) ->
     {error, {error_removing_from_org_user_group, Error}}.
 
-deprovision_fetch_org_global_admins(ok, #context{authz_context = AuthzContext,
-                                                 org_name = OrgName} = Context) ->
-    Result = oc_chef_authz_db:fetch_global_group_authz_id(AuthzContext, OrgName, "global_admins"),
-    deprovision_remove_global_org_admin_ace(Result, Context) ;
-deprovision_fetch_org_global_admins(Error, Context) ->
+deprovision_fetch_org_read_access_group(ok, #context{authz_context = AuthzContext,
+                                                     org_name = OrgName} = Context) ->
+    Result = oc_chef_authz_db:fetch_read_access_group(AuthzContext, OrgName),
```

Comment:
Thanks for the careful review.  This was only working because the -ifdef around an export_all was commented out:

https://github.com/chef/chef-server/blob/ssd/global-admin-group-rename/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_db.erl#L38-L40